### ISSUE TITLE
Embed video on homepage

### ISSUE THREAD

**Issue by kgish** [kgish/discourse-pvda-plugin#6]
https://www.dropbox.com/s/bn107o990pz4xt1/Ledendemocratie_def_new.mov?dl=0

**Comment by kgish** [comment]
@Socioneers : Where exactly would you like to place the video on the homepage and how large should it be?

**Comment by kgish** [comment]
could be with this solution https://github.com/scossar/whitelist-iframe

..op https://www.youtube.com/watch?v=fTNP6UVRx64&feature=youtu.be

embedded:
<iframe width="560" height="315" src="https://www.youtube.com/embed/fTNP6UVRx64" frameborder="0" allowfullscreen></iframe>

**Comment by Socioneers** [comment]
Hi Kiffin,
Could you look in to this issue.
Thanks

**Comment by kgish** [comment]
Yes, please tell me where you want this video to appear as well as the desired 
dimensions.

Something like the pvda website?

http://www.pvda.nl/berichten/2016/09/One+man%2C+one+vote

**Comment by Socioneers** [comment]
See my 2 examples, dimensions are not specified.

Raoul
---

+31(0)6 4 66 28 28 3
@socioneers
http://socioneers.com


This email is hosted sustainably.
Important Note: Any unauthorised copying, disclosure or distribution of the material inclosed is strictly forbidden. The Socioneers are registered at the Dutch Chamber of Commerce under number: KVK 09178876

**Comment by kgish** [comment]
Please attach examples to this issue for easier reference.

**Comment by Socioneers** [comment]
![screen shot 2016-09-09 at 11 01 47](https://cloud.githubusercontent.com/assets/11832800/18382052/a9c16ab0-767f-11e6-8842-4e16ea2bc38b.jpg)
![screen shot 2016-09-09 at 11 01 48](https://cloud.githubusercontent.com/assets/11832800/18382053/a9c3b82e-767f-11e6-8276-24aa688cfbcb.jpg)

**Comment by Socioneers** [comment]
I think on the meta forum there are examples from people that pasted video's in the blue text box of the second example. I think that is the best option.

**Comment by Socioneers** [comment]
sorry to change plans but could you work on the urgent things. I put some new things there.

Thanks

**Comment by kgish** [comment]
Sure, no problem. I see a number of urgent stuff now, please tell me in what order they need to be fixed.
